click/_compat.py

```diff
@@ -482,9 +482,15 @@ def open_stream(filename, mode='r', encoding=None, errors='strict',
 
     # Non-atomic writes directly go out through the regular open functions.
     if not atomic:
-        if encoding is None:
-            return open(filename, mode), True
-        return io.open(filename, mode, encoding=encoding, errors=errors), True
+        open_kwargs = {}
```

Comment:
This needs to happen in atomic mode too, I'll move the kwargs outside the `if` block.